src/components/Checkout/PaymentWrapper.vue

```diff
@@ -19,7 +19,7 @@
 			</div>
 			<div>
 				<braintree-checkout
-					v-show="selectedOption === 'bt'"
+					v-if="selectedOption === 'bt'"
```

Comment:
We could potentially go either way with this one, this setup ensure that each component is only initialized when it's needed.

Reply:
That makes sense, no need to initialize unless we need it. 👍